canonical-web-and-design/react-components
# Add Navigation component

## Done

- Add a component to handle the Vanilla navigation pattern (originally built in MAAS here: https://github.com/canonical-web-and-design/maas-ui/pull/3833).

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Go to the [navigation page](https://react-components-767.demos.haus/?path=/docs/navigation--default-story) in Storybook and check that the docs make sense and the examples work as expected.

## Fixes

Fixes: #134.
